pkg/apiserver/api/v1/instance.proto

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2019 The KubeCarrier Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+package kubecarrier.api.v1;
+option go_package = "v1";
+
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+import "meta.proto";
+
+message Instance {
+  ObjectMeta metadata = 1;
+  string offering = 2;
+  string spec = 3;
```

Comment:
Could you please add a wrapper object for arbitrary data? (`status` and `spec` fields)
I would like to have the possibility to support multiple encoding styles the future.
e.g. `JSON` or `YAML` - maybe even nested protobuf

```
message RawObject {
  string encoding = 1;
  bytes data = 2;
}
```
Please use `JSON` as default for know, although I already see a use-case for the UI requesting the nested object as `YAML` (When editing the YAML directly)

Reply:
done